extmod/vfs_blockdev.c

```diff
@@ -46,31 +46,37 @@ void mp_vfs_blockdev_init(mp_vfs_blockdev_t *self, mp_obj_t bdev) {
     }
 }
 
+// Helper function to minimise code size of read/write functions
+static int mp_vfs_blockdev_call_rw(int n_args, mp_obj_t *args, size_t block_num, size_t block_off, size_t len, void *buf) {
```

Comment:
I suggest making `n_args` of type `size_t` to match `mp_call_method_n_kw`.

Also maybe putting it at the end of the argument list, to reduce code size (see other comment).